da-65-anonymusquaestions-projectfile.xml

```diff
@@ -3,20 +3,17 @@
 <listofFileNames>
   <header>
     <authorName>Anonymous</authorName>
-    <commentaryName>Quaestiones De anima I-II</commentaryName>
+    <commentaryName>Quaestiones De anima I--III</commentaryName>
     <commentaryid>da-65</commentaryid>
     <commentaryslug>da-65</commentaryslug>
     <authorUri>http://scta.info/resource/Anonymous</authorUri>
     <parentUri>http://scta.info/resource/da-65</parentUri>
     <parentWorkGroup>http://scta.info/resource/deanima</parentWorkGroup>
     <textfilesdir>/Users/jcwitt/Projects/scta/scta-texts/da-65/</textfilesdir>
     <questionListSource></questionListSource>
-    <questionListOriginalEditor>Sten Ebbesen</questionListOriginalEditor>
-    <!-- I received the list of questions from SE -->
+    <questionListOriginalEditor>Michael Stenskjær Christensen</questionListOriginalEditor>
     <questionListEncoder>Michael Stenskjær Christensen</questionListEncoder>
-    <!-- possible metadata should go here or be added elsewhere
-    -->
-    <gitRepoBase>https://github.com/mldac/</gitRepoBase>
+    <gitRepoBase>https://github.com/scta-texts/</gitRepoBase>
```

Comment:
fyi, gitRepoBase needs the attribute `@type=toplevel` to know that we are creating one repo for the entire top level expression rather than a repo for each item level expression. I'll accept the PR and then make the change. 

We can change this to the default behavior in future versions and then only require the @type=itemlevel when we want repos for each item level.